### DISCUSSION TITLE
Instructions on installing Ray on remote Laravel Vapor servers.

### DISCUSSION THREAD

**Discussion by goldzulu** [spatie/ray#602] [Q&A]
Hi, 

I can't seem to find any instructions on installing Ray onto Laravel Vapor deployment. There are bits and pieces that I saw about it being fixed so perhaps there is a way to make it work but I am not sure how. I have a docker based deployment of Laravel Vapor if that helps.

Many thanks in advance and apologies if this a duplicate question.

Regards
Goldzulu

**Comment by freekmurze** [comment]
I'm thinking it's impossible now to use Ray on Vapor as there is no way for the Vapor app to communicate with Ray. For remote connections we rely on SSH, which simply isn't available.

**Comment by goldzulu** [comment]
> I'm thinking it's impossible now to use Ray on Vapor as there is no way for the Vapor app to communicate with Ray. For remote connections we rely on SSH, which simply isn't available. 

Yes that was my concern actually when I saw the part on how to communicate with the server. I thought there was some workarounds. It's such a shame as I love using it via sail on local dev!

I am using inertiajs and vue3 is stack. Could I slingshot the comms through them? Also I do have a jump box with SSH capability that communicate to the the vapor environments. Would that help?
